database/index.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017/moodu');
+
+
+/* SCHEMAS */
+let reviewsSchema = mongoose.Schema({

```

Comment:
I don't think I've ever defined a Mongoose Schema like this before. https://mongoosejs.com/docs/guide.html#definition